src/components/Form/FileUpload/FileUpload.module.scss

```
@@ -70,6 +69,10 @@
   .file-upload-title-mandatory {
     color: var(--color-red500);
   }
+
+  & {
+    @include transition(all, 0.2s, ease-in-out);
+  }
```

can't we just do:
```suggestion
    @include transition(all, 0.2s, ease-in-out);
```
It will be less confusing IMO